tracing/tracer.go

```diff
@@ -37,25 +27,67 @@ func (tp *TestTraceProvider) Tracer(name string) trace.Tracer {
 var Configured bool
 
 type Config struct {
+	ServiceName string            `long:"service-name"  description:"otlp service name" default:"concourse-web"`
+	Attributes  map[string]string `long:"attributes"  description:"otlp attributes to attach to traces as metadata"`
```

Comment:
usually these go-flags map flags work best where the flag name itself is singular, since they can be specified multiple times; e.g. `concourse web --tracing-attribute foo:bar --tracing-attribute baz:qux`. also not otlp-specific
```suggestion
	Attributes  map[string]string `long:"attribute"  description:"attributes to attach to traces as metadata"`
```